### ISSUE TITLE
Point edit this page button to different branch?

### ISSUE THREAD

**Issue by gregdegruy** [vsoch/docsy-jekyll#40]
Is this possible?
For example when I click that button it brings me to `https://github.com/vsoch/docsy-jekyll/edit/master/_docs/getting-started.md`, but I would like to point it to a different branch like `https://github.com/vsoch/docsy-jekyll/edit/main/_docs/getting-started.md`.

![image](https://user-images.githubusercontent.com/8151045/109948272-028e9b80-7c8f-11eb-84ed-0983ad6227b7.png)

**Comment by vsoch** [comment]
Yes! You'll want to edit the links in this file: https://github.com/vsoch/docsy-jekyll/blob/master/_includes/editable.html

**Comment by bhitney** [comment]
I modified the template to use a branch variable on the _config.yml so branch can be specified (only tested with github pages). I'll submit a pull request if you are interested in merging that change.
